target/linux/pistachio/base-files/lib/functions/bluetooth.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+
+elapsed_time() {
```

Comment:
This file contains the function which is just used by the 2 Bluetooth scripts.. so al the Bluetooth related functions in the future profiles can reside here. I am ok to changing the name of the script